provider/aws/templates/resource/elasticsearch.tmpl

```diff
@@ -0,0 +1,128 @@
+{{ define "resource" }}
+{
+  "AWSTemplateFormatVersion" : "2010-09-09",
+    "Conditions": {
+    },
+    "Parameters": {
```

Comment:
Would you please alphabetize these parameters? Makes it easier to manage as things grow.